.github/workflows/task_fe_unit_tests.yml

```diff
@@ -45,7 +45,10 @@ jobs:
       - name: Run unit tests
         working-directory: ./frontend/app
         run: pnpm run --filter rotki test:unit
+      - name: Verify External Links
+        working-directory: ./frontend/app
+        run: pnpm run link:verify
```

Comment:
This Im not sure where to put it

Reply:
except for this, which @kelsos can talk about, others lgtm!

Reply:
Can we make it so that it only runs on nightlies?